### ISSUE TITLE
<SanityContent> Duplicate keys detected

### ISSUE THREAD

**Issue by ovsw** [nuxt-community/sanity-module#116]
For any block content from sanity, if the block content contains `<strong>`  marks (bold text) Nuxt will throw a warning in development (browser console):
```
vue.runtime.esm.js?2b0e:619 [Vue warn]: Duplicate keys detected: ''. This may cause an update error.

found in

---> <SanityContent>
```
In my component, all I do is:
```
<SanityContent  :blocks="blocktext"  />
```
where `blocktext` is the block content from Sanity.
The text renders ok in the front-end though.

Why is this happening? Am I doing something wrong?

### Version
module:  0.9.0 (and previously 0.7.1)
nuxt: 2.15.5

**Comment by TimHill83** [comment]
I'm getting this too.  Is it a bug in the way it's rendered, or is there a way round it?

**Comment by ovsw** [comment]
maybe should have marked this as a bug, I really don't see any way of using `<SanityContent/>` that doesn't cause the errors.

**Comment by filippolcr** [comment]
@danielroe  @ovsw  @TimHill83 These errors are shown only with yarn dev.

![image](https://user-images.githubusercontent.com/13961934/117954367-3e8d4f00-b317-11eb-9472-ddd328e3e9dc.png)

Not detected with yarn generate && yarn start with target static.

"nuxt": "2.15.4" and  "@nuxtjs/sanity": "0.9.2"

**Comment by danielroe** [comment]
Thanks! Should be fixed in v0.9.4 ❤️

**Comment by TimHill83** [comment]
Many thanks. Keep up the good work!

Sent from my phone. Apologies in advance for brevity and any typos.

________________________________
